Changelog.rst

```diff
@@ -59,8 +65,22 @@ auto-update for all future RPM updates. If you wish to disable this behavior,
 you should edit the options in ``/etc/simp/adapter_config.yaml``.
 
 .. NOTE::
-   Anything that is in a Git or Subversion repository in the `simp` environment
-   will **not** be overwritten by ``simp_rpm_helper``.
+   Anything that is in a Git or Subversion repository in the ``simp`` environment
```

Comment:
[SIMP-2261](https://simp-project.atlassian.net/browse/SIMP-2261) should block this.